src/Menu/index.js

```diff
@@ -0,0 +1,22 @@
+import React, { PropTypes } from 'react';
+import styles from './style.scss';
+
+export { default as MenuItem } from './MenuItem.js';
+
+const Menu = ({ children, bordered }) => (
+  <ul className={`${styles.list} ${bordered && styles.bordered}`}>
```

Comment:
allow custom classNames, and other props?
```
 <ul className={`${styles.list} ${bordered && styles.bordered} ${className}`} {...props} >
```

thoughts on allowing this escape hatch?